tests/pipeline/test_international_markets.py

```diff
@@ -402,6 +402,16 @@ def test_explicit_specialization_matches_implicit(self):
             sessions[-1],
         )
 
+    def test_cannot_convert_volume_data(self):
+        with self.assertRaises(TypeError) as exc:
+            Pipeline({'volume': EquityPricing.volume.fx('EUR').latest})
```

Comment:
```suggestion
             EquityPricing.volume.fx('EUR')
```

We don't need to construct a Pipeline around this for the error to trigger, so no reason to include that in the code under test.